modular_skyrat/modules/importedvendors/code/vendor_snacks.dm

```diff
@@ -0,0 +1,275 @@
+// Snacks and drinks for the 'snacks' tab of vendors
+
+//Yangyu Stuff
+
+/obj/item/reagent_containers/cup/glass/dry_ramen/prepared
+	name = "cup ramen"
+	desc = "This one even comes with water, amazing!"
+	list_reagents = list(/datum/reagent/consumable/hot_ramen = 15, /datum/reagent/consumable/salt = 3)
+
+/obj/item/reagent_containers/cup/glass/dry_ramen/prepared/hell
+	name = "spicy cup ramen"
+	desc = "This one comes with water, AND a security checkpoint's worth of capsaicin!"
+	list_reagents = list(/datum/reagent/consumable/hell_ramen = 15, /datum/reagent/consumable/salt = 3)
+
+/obj/item/food/vendor_snacks
+	name = "\improper God's Strongest Snacks"
+	desc = "You best hope you aren't a sinner. (You should never see this item please report it)"
+	icon = 'modular_skyrat/modules/importedvendors/icons/imported_quick_foods.dmi'
+	icon_state = "foodpack_generic"
+	trash_type = /obj/item/trash/vendor_trash
+	bite_consumption = 10
+	food_reagents = list(/datum/reagent/consumable/nutriment = INFINITY)
+	junkiness = 10
+	custom_price = PAYCHECK_LOWER * INFINITY
+	tastes = list("the unmatched power of the sun" = 10)
+	foodtypes = JUNKFOOD | CLOTH | GORE | NUTS | FRIED | FRUIT //You don't want to know what's in the broken debug snacks
+	w_class = WEIGHT_CLASS_SMALL
+
+/obj/item/trash/vendor_trash
+	name = "\improper God's Weakest Snacks"
+	desc = "The leftovers of what was likely a great snack in a past time."
+	icon = 'modular_skyrat/modules/importedvendors/icons/imported_quick_foods.dmi'
+	icon_state = "foodpack_generic_empty"
+	custom_materials = list(/datum/material/plastic=1000)
+
+/obj/item/food/vendor_snacks/ricecrackers
+	name = "rice crackers"
+	desc = "Despite most of the package being clear, you will never truly know what flavor these are until you eat them."
+	icon_state = "ricecracka"
+	trash_type = /obj/item/trash/vendor_trash/ricecracka
+	food_reagents = list(/datum/reagent/consumable/nutriment = 2, /datum/reagent/consumable/rice = 2)
+	tastes = list("incomprehensible flavoring" = 1, "rice cracker" = 2)
+	foodtypes = JUNKFOOD | GRAIN
+	custom_price = PAYCHECK_LOWER * 0.8
+
+/obj/item/food/vendor_snacks/ricecrackers/MakeLeaveTrash()
+	AddElement(/datum/element/food_trash, trash_type, FOOD_TRASH_POPABLE)
+
+/obj/item/trash/vendor_trash/ricecracka
```

Comment:
I'd personally make sure this matches with the vendor_snacks version just for future-proofing; searching the files for "Ricecrackers" would make you miss "Ricecracka"